homeassistant/components/google_assistant_sdk/__init__.py

```diff
@@ -154,6 +156,12 @@ async def async_process(
         self, user_input: conversation.ConversationInput
     ) -> conversation.ConversationResult:
         """Process a sentence."""
+        try:
+            google_assistant_sdk_extension = importlib.import_module(
+                "custom_components.google_assistant_sdk_extension"
+            )
```

Comment:
This PR put custom integration requirements into our Core codebase, which isn't something we want.

The reason is that well, someone makes another custom component for google assistant and this becomes a list...

And even if we did this, we potentially get our codebase riddled with this kind of exceptions (which is very hard to maintain).